daffodil-cli/src/it/scala/org/apache/daffodil/debugger/TestCLIDebugger.scala

```diff
@@ -934,6 +958,8 @@ class TestCLIdebugger {
       shell.sendLine("continue")
       shell.expect(contains("<f xmlns=\"\">2</f>"))
       shell.sendLine("quit")
+
```

Comment:
I think the quit here needs to be removed. I think it was accidentally added back when conflicts were fixed.